### PR TITLE
Migrate galaxy_sequence_utilities to IUC

### PR DESCRIPTION
**Pull request by ic4f**
This commit *does not* include any modifications to the tools.
(fastq_groomer revision to be committed separately)

All .shed files have been updated:
- owner switched from devteam to iuc
- remote_repo_url updated to tools-iuc
- keys reordered to be consistent with other .shed files in iuc repo:
  categories, desc, long_desc, name, owner, remote_rep_url, type

Related discussion: https://github.com/galaxyproject/tools-devteam/pull/521#issuecomment-504415399

Ping @bgruening 

FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tool_collections/galaxy_sequence_utils/fastq_combiner/.shed.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+categories:
+- Fastq Manipulation
+- Fasta Manipulation
+description: Combine FASTA and QUAL into FASTQ
+long_description: |
+  This tool joins a FASTA file to a Quality Score file, creating
+  a single FASTQ block for each read.
+name: fastq_combiner
+owner: iuc
```
Comment (bgruening) [inline tool_collections/galaxy_sequence_utils/fastq_combiner/.shed.yml]: If this was devteam, it should stay devteam. The reason here is that a tool-id in Galaxy is TS+owner+tool-id. And we do not want to break this id by changing the owner.

Comment (ic4f) [reply tool_collections/galaxy_sequence_utils/fastq_combiner/.shed.yml]: Fixed. Thank you for explaining this.

**Comment by ic4f** [comment]
Just an update. This is not abandoned, still waiting on a code review for this: https://github.com/galaxyproject/sequence_utils/pull/23 .

**Comment by ic4f** [comment]
This should work, I think. Did I update the versions correctly? (see note in last commit) @mvdbeek 
I'll move these tools from devteam-tools to devteam-tools/legacy after this is approved+merged.

**Comment by ic4f** [comment]
Oh great! Linting errors. Fixing....

**Comment by bgruening** [comment]
Restarted.

**Comment by ic4f** [comment]
Thank you! Some legitimate and consistent errors have showed up. Fixing.

File: tool_collections/galaxy_sequence_utils/fastq_manipulation/t.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+TARGET_TOOL_DIRS=("tools/fastq_trimmer_by_quality" "tool_collections/galaxy_sequence_utils/fastq_combiner" "tool_collections/galaxy_sequence_utils/fastq_manipulation" "tool_collections/galaxy_sequence_utils/fastq_groomer" "tool_collections/galaxy_sequence_utils/fastq_filter")
```
Comment (nsoranzo) [inline tool_collections/galaxy_sequence_utils/fastq_manipulation/t.sh]: I think this file can be removed?

File: tool_collections/galaxy_sequence_utils/fastq_paired_end_interlacer/fastq_paired_end_interlacer.xml [hunk 3]
```diff
@@ -0,0 +1,141 @@
+<tool id="fastq_paired_end_interlacer" name="FASTQ interlacer" version="@TOOL_VERSION@">
```
Comment (nsoranzo) [inline tool_collections/galaxy_sequence_utils/fastq_paired_end_interlacer/fastq_paired_end_interlacer.xml]:
```suggestion
<tool id="fastq_paired_end_interlacer" name="FASTQ interlacer" version="1.2.0.1+galaxy0">
```

File: tool_collections/galaxy_sequence_utils/fastq_paired_end_joiner/fastq_paired_end_joiner.xml [hunk 4]
```diff
@@ -0,0 +1,108 @@
+<tool id="fastq_paired_end_joiner" name="FASTQ joiner" version="2.0.1.2">
```
Comment (nsoranzo) [inline tool_collections/galaxy_sequence_utils/fastq_paired_end_joiner/fastq_paired_end_joiner.xml]:
```suggestion
<tool id="fastq_paired_end_joiner" name="FASTQ joiner" version="2.0.1.1+galaxy0">
```

**Comment by mvdbeek** [comment]
Alright, permissions are set.

File: tool_collections/galaxy_sequence_utils/fastq_manipulation/fastq_manipulation.xml [hunk 5]
```diff
@@ -0,0 +1,443 @@
...
+    </edam_topics>
+    <edam_operations>
+        <edam_operation>operation_0233</edam_operation>
+    </edam_operations>
+    <command><![CDATA[
+gx-fastq-manipulation '$input_file' '$fastq_manipulation_file' '$output_file' '$output_file.files_path' '${input_file.extension[len('fastq'):]}'
+    ]]></command>
+    <configfiles>
+        <configfile name="fastq_manipulation_file"><![CDATA[##create an importable module
+#import binascii
```
Comment (nsoranzo) [inline tool_collections/galaxy_sequence_utils/fastq_manipulation/fastq_manipulation.xml]:
```suggestion
```

File: tool_collections/galaxy_sequence_utils/fastq_masker_by_quality/fastq_masker_by_quality.xml [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+    </macros>
+    <expand macro="requirements"/>
+    <edam_topics>
+        <edam_topic>topic_0622</edam_topic>
+    </edam_topics>
+    <edam_operations>
+        <edam_operation>operation_0368</edam_operation>
+    </edam_operations>
+    <command><![CDATA[
+gx-fastq-masker-by-quality '$input_file' '$output_file' -f '${input_file.extension[len( 'fastq' ):]}' -s ${quality_score} -c ${score_comparison}
```
Comment (nsoranzo) [inline tool_collections/galaxy_sequence_utils/fastq_masker_by_quality/fastq_masker_by_quality.xml]:
```suggestion
gx-fastq-masker-by-quality '$input_file' '$output_file' -f '${input_file.extension[len('fastq'):]}' -s ${quality_score} -c ${score_comparison}
```

File: tool_collections/galaxy_sequence_utils/fastq_trimmer/fastq_trimmer.xml [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+    </macros>
+    <expand macro="requirements"/>
+    <edam_topics>
+        <edam_topic>topic_0622</edam_topic>
+    </edam_topics>
+    <edam_operations>
+        <edam_operation>operation_3192</edam_operation>
+    </edam_operations>
+    <command><![CDATA[
+gx-fastq-trimmer '$input_file' '$output_file' ${offset_type['left_column_offset']} ${offset_type['right_column_offset']} ${offset_type['base_offset_type']} '${input_file.extension[len('fastq'):]}' $keep_zero_length
```
Comment (nsoranzo) [inline tool_collections/galaxy_sequence_utils/fastq_trimmer/fastq_trimmer.xml]:
```suggestion
gx-fastq-trimmer '$input_file' '$output_file' ${offset_type.left_column_offset} ${offset_type.right_column_offset} ${offset_type.base_offset_type} '${input_file.extension[len('fastq'):]}' $keep_zero_length
```

File: tool_collections/galaxy_sequence_utils/fastq_manipulation/fastq_manipulation.xml (hunk 5 above)
Comment (mvdbeek) [reply tool_collections/galaxy_sequence_utils/fastq_manipulation/fastq_manipulation.xml]: We'll need that for everything inside ${}

**Comment by mvdbeek** [comment]
Thanks everyone, that was a good team effort!

**Pull request merged**